travis.sh

```diff
@@ -3,11 +3,18 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+case $OSTYPE in darwin*)
+    brew update > /dev/null
+    brew upgrade cmake
```

Comment:
I can see in this PR's history that upgrading CMake fixed dome issues I had on OSX travis build. Doesn't meter now as I'll probably drop this PR.